### PR TITLE
Feature/link importer

### PR DESCRIPTION
**Pull request by breneser**

### CODE REVIEW

File: web/domains/case/access/actions.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import structlog as logging
+from django.contrib import messages
+from django.shortcuts import redirect
+from django.urls import reverse
+
+from web.domains.case.access.models import AccessRequestProcess
+from web.views.actions import PostAction
+
+# from viewflow.decorators import flow_func
```
Comment (francodacosta) [inline web/domains/case/access/actions.py]: should this be deleted?

File: web/utils/utils.py [hunk 2]
```diff
@@ -80,6 +47,7 @@ class FilevalidationService():
         'wsc',
         'wsf',
         'wsh',
+        'txt'
```
Comment (francodacosta) [inline web/utils/utils.py]: this should be removed, we allow for .txt files

Comment (breneser) [reply web/utils/utils.py]: Oh, guess merge wasn't as right as I though :D I will fix this

File: web/domains/case/access/actions.py (hunk 1 above)
Comment (breneser) [reply web/domains/case/access/actions.py]: the comment? Or all  those imports?

Comment (francodacosta) [reply web/domains/case/access/actions.py]: the comment

**Pull request merged**